### PR TITLE
fix pip numbers leakage

### PR DESCRIPTION
**Pull request by alexcg1**
Fixes https://github.com/jina-ai/internal-tasks/issues/86

At present, anything installed from `requirements.txt` in examples repo is not tracked by pypi, so we can't get those metrics. But do we even want those numbers?

- If it's our CI doing that, we DON'T want those numbers. 
- If it's our users, we DO want those numbers. 
 
This PR creates a separate requirements.txt in each example's `test/` directory, and tells our CI to install from that. 

It reverts `./requirements.txt` to installing from pypi, which is what real users would do.

- test: separate reqs for users and ci. users install direct
- test: set ci to install requirements.txt from test dir

### CODE REVIEW

**Comment by alexcg1** [comment]
Thank god for vim's automation via `.`. That saved me a lot of work!

File: advanced-vector-search/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/jina-ai/jina.git@v1.2.0#egg=jina[hub, http]
+jina[hub, http]
```
Comment (nan-wang) [inline advanced-vector-search/requirements.txt:1]:
```suggestion
jina[hub, http]==1.2.0
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
We need to fix the version

File: advanced-vector-search/requirements.txt (hunk 1 above)
Comment (alexcg1) [reply advanced-vector-search/requirements.txt]: Fixed for all

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**